app/controllers/shopify_app/callback_controller.rb

```diff
@@ -78,6 +78,10 @@ def token
       auth_hash['credentials']['token']
     end
 
+    def shop_scopes
+      auth_hash['extra']['scope'].split(',')
```

Comment:
Could `auth_hash['extra']['scope']` ever be nil here?

Reply:
Only if there are no scopes associated with a shopify_session (shop/user). I guess we can add a check for that as well